operations/network/management_network/bican_support_matrix.md

```diff
@@ -0,0 +1,113 @@
+# BICAN Support Matrix - Shasta Customer Access Networks
+
+## Data sheet
+
+### Shasta networking
+
+### Customer Access Networks
+
+### Overview
+
+Customer Access Networks provide the interface between Shasta system networking and the customer site network.  Customer Access Networks (CANs) are routed networks with broadcast domain separation.  Customer Access
+Networks provide higher availability and more flexibility in accessing cloud services compared to traditional "bastion hosts", and are more in line with cloud-native architecture of Shasta as whole.
+
+Customer Access Networks (CANs) provide flexible networking at the edge between the site and Shasta system to do the following:
+
+* Perform administrative tasks on the system.
+* Run jobs and move job data to and from the system.
+* Access site resources like DNS and LDAP from the system.
+
+## Feature access matrix
+
+For CSM 1.2, the notion of the CAN has been expanded to meet customer requests for increased flexibility and policy control.
+
+|     |                            |                              |                            |                                  |
+| --- |----------------------------|------------------------------|----------------------------|----------------------------------|
+|     |                            | **User Access Jobs**         | **User Access Jobs**      | **Management or Administrators** |
```

Comment:
If these are the column headers, I believe they should be on the first line of the table, above the horizontal dashes. And in that case, they will not need the `**` markups, because they will automatically be bolded.